### ISSUE TITLE
Not able to generate the Allure mocha reports while running tha mocha tests parallel. An error occurred "MochaAllure: "label" can't be used in parallel mode! To use Allure Runtime API, please, switch back to single thread mode"

### ISSUE THREAD

**Issue by islam-tenovos** [allure-framework/allure-js#651]
**Describe the bug**
Facing an error message while try to run the mocha tests parallel. Error **"MochaAllure: "label" can't be used in parallel mode! To use Allure Runtime API, please, switch back to single thread mode" & "MochaAllure: "attachment" can't be used in parallel mode! To use Allure Runtime API, please, switch back to single thread mode."**. 

**To Reproduce**
1. Create 2 mocha tests.
2. configure the allure-mocha as reporter in config.json.
3. Define the code to generate the Allure steps and description. 
4. Call the code to generate report in the end of execution. 

**Expected Behavior**
Both test cases should run successfully and allure description and steps should be created according to attached screenshot.

**Screenshots**
![expected result_allure](https://user-images.githubusercontent.com/73897048/229546243-03dd21de-2fec-435b-bac5-04f5fa8f3d4b.jpg)

**Desktop (please complete the following information):**
 - OS: [win10]
 - Browser [chrome]
 - Version [111.0.5563.147]
 
**Command to run tests**
npm run local:Test
[mocha-project.zip](https://github.com/allure-framework/allure-js/files/11139609/mocha-project.zip)

**Comment by baev** [comment]
duplicate of https://github.com/allure-framework/allure-js/issues/619
